### PR TITLE
Darwin's Danger Shield

### PR DESCRIPTION
**Pull request by StarInAPond**
Deletes weird undocumented fire resistance removal, adds +40 greater jump height allowing sniper to jump on most small props

### CODE REVIEW

**Comment by woisalreadytaken** [comment]
I shouldn't have much say here but more wallclimbs giga cringe

**Comment by StarInAPond** [comment]
Agreed, this could be too good, changed to once used +40% (or 25?) jump height

File: addons/sourcemod/configs/vsh/vsh.cfg [hunk 1]
```diff
@@ -1860,8 +1860,9 @@
 		
 		"231"	//Darwin's Danger Shield
 		{
-			"desp"			"Darwin's Danger Shield: {positive}No fall damage"
-			"attrib"		"60 ; 1.0 ; 527 ; 0.0 ; 275 ; 1.0"
+			"desp"			"Darwin's Danger Shield: {positive}No fall damage, +40% greater jump height"
+			"attrib"		"275 ; 1.0 ; 326 ; 1.4"
+
```
Comment (FortyTwoFortyTwo) [inline addons/sourcemod/configs/vsh/vsh.cfg:1865]: Extra space

**Review by FortyTwoFortyTwo — COMMENTED** [review]
Think Darwin used to have +25% extra jump height years ago, don't remember full story behind but might be okay to bring it back

**Review by FortyTwoFortyTwo — APPROVED** [review]

**Pull request merged**